src/test/java/tests/RegisterTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import common.Constant;
+import helper.DataHelper;
+import helper.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_object.RegisterPage;
+
+public class RegisterTest extends BaseTest {
+    RegisterPage registerPage = new RegisterPage();
+    String email = "";
```

Comment:
you don't need to specify here, you can specify in the test.

Reply:
fixed